### DISCUSSION TITLE
Does reloading the Cache update SAML settings

### DISCUSSION THREAD

**Discussion by timlegge** [kimai/kimai#2244] [General]
Hi

We have to update the Identity Provider Certificate. in local.conf.  If I change that setting and run 

bin/console kimai:reload --env=prod

will the new certificate be used?

Tim

**Comment by timlegge** [comment]
Answering my own question so I will remember next time:

Yes, reloading the cache reloads the SAML settings from local.yaml.  

I changed the local.yaml as the root user but made sure to reload the cache as the user running kimai in the docker container in step 8

In a docker container you need to:

1. sudo docker exec --user=root -it kimai2_kimaiapp_1 /bin/bash
2. cd /opt/kimai/
3. vi config/packages/local.yaml
4. locate x509cert:
5. 5.	Update: x509cert : 
6. Save
7. exit
8. sudo docker exec -it kimai2_kimaiapp_1 /bin/bash
9. cd /opt/kimai
10. bin/console cache:clear
11. exit

**Reply by kevinpapst** [reply]
Sorry for missing the question and thanks for sharing your solution!
